charts/mattermost-enterprise-edition/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Mattermost Enterprise server with high availitibity.
 name: mattermost-enterprise-edition
-version: 1.1.1
+version: 1.1.2
```

```suggestion
version: 1.2.0
```
since is not a bug fix